src/bun.js/webcore/response.zig

```diff
@@ -619,23 +621,32 @@ pub const Fetch = struct {
         javascript_vm: *VirtualMachine = undefined,
         global_this: *JSGlobalObject = undefined,
         request_body: HTTPRequestBody = undefined,
+        // buffer being used by AsyncHTTP
```

Comment:
use `///` because `//` doesn't show up in autocomplete

Reply:
fixed https://github.com/oven-sh/bun/pull/4127/commits/4676cfe8c877ca99e2fb4f2598afb51b51ceaadc